pom.xml

```diff
@@ -1761,6 +1761,8 @@
                         <runOnlyOnce>true</runOnlyOnce>
                         <injectAllReactorProjects>true</injectAllReactorProjects>
                         <offline>true</offline>
+                        <!-- https://github.com/git-commit-id/git-commit-id-maven-plugin/issues/215 -->
```

Comment:
Add description like "Required to make build work in a Git worktree. See (link)"

Reply:
Done, thanks!